config/initializers/rolify.rb

```diff
@@ -0,0 +1,7 @@
+Rolify.configure do |config|
+  # Dynamic shortcuts for User class (user.is_admin? like methods). Default is: false
```

Comment:
We should either uncomment stuff or delete the block. Just `Rolify.configure` would accomplish exactly the same.